source/common/config/well_known_names.cc

```diff
@@ -238,6 +238,13 @@ TagNameValues::TagNameValues() {
 
   // grpc.(<stat_prefix>).**
   addTokenized(GOOGLE_GRPC_CLIENT_PREFIX, "grpc.$.**");
+
+  // listener.[<address>.]ssl.certificate.(<cert_name>).expiration_unix_time_seconds or
+  // cluster.[<cluster_name>.]ssl.certificate.(<cert_name>).expiration_unix_time_seconds
+  addRe2(
+      TLS_CERTIFICATE,
+      R"(^<LISTENER_OR_CLUSTER_WITH_NAME>\.ssl\.certificate(\.(<TAG_VALUE>)\.expiration_unix_time_seconds)$)",
```

Comment:
In case we ever add another stat under this hierarchy (maybe usage count of a certificate for server certs?):
```suggestion
      R"(^<LISTENER_OR_CLUSTER_WITH_NAME>\.ssl\.certificate(\.(<TAG_VALUE>)\..*)$)",
```

Reply:
sg, updated